src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/EqualsAvoidNullCheck.java

```diff
@@ -68,9 +68,27 @@
  * <pre>
  * &lt;module name=&quot;EqualsAvoidNull&quot;/&gt;
  * </pre>
+ * <p>
+ * To configure the check for ignoreEqualsIgnoreCase:
```

Comment:
`To configure the check for ignoreEqualsIgnoreCase` -> To configure the check to ignore 'equalsIgnoreCase' method